helix-core/Cargo.toml

```
@@ -18,8 +18,11 @@ integration = []
 [dependencies]
 helix-stdx = { path = "../helix-stdx" }
 helix-loader = { path = "../helix-loader" }
+helix-event = { path = "../helix-event" }
 helix-parsec = { path = "../helix-parsec" }
 
+filesentry = "0.2.1"
+ignore = "0.4"
```

```suggestion
ignore.workspace = true
```

now that we have the workspace dep on ignore